web/impact/impact/v1/helpers/profile_helper.py

```diff
@@ -23,10 +27,45 @@
     MemberProfile,
     ProgramFamily,
 )
-from impact.models.base_profile import (
-    PHONE_MAX_LENGTH,
-    TWITTER_HANDLE_MAX_LENGTH,
-)
+
+
+EXPERT_BOOLEAN_FIELD = {
+    "json-schema": {
+        "type": "boolean",
+    },
+    "GET": {
+        "included": "could_be_expert",
+        "description": "This field exists only when user_type is 'expert'",
```

Comment:
The description value should be extracted to a CONSTANT and assigned to the places where it makes sense, so it's more DRY.

Reply:
Done.